src/testrel.jl

```diff
@@ -106,32 +127,30 @@ end
 const AcceptedSourceTypes =
     Union{String, Pair{String, String}, Vector{String}, Dict{String, String}}
 
-convert_to_install_kv(install_dict::Dict{String, String}) = install_dict
-convert_to_install_kv(install_pair::Pair{String, String}) = Dict(install_pair)
-convert_to_install_kv(install_string::String) = convert_to_install_kv([install_string])
-function convert_to_install_kv(install_vector::Vector{String})
-    models = Dict{String, String}()
-    for i in enumerate(install_vector)
-        models["test_install" * string(i[1])] = i[2]
-    end
-    return models
-end
-
 """
-    Transaction Step used for `test_rel`
-
-    - `install::Dict{String, String}`:
-        sources to install in the database.
+Transaction Step used for `test_rel`
 
-    - `broken::Bool`: if the computed values are not currently correct (wrt the `expected`
+  - `query::String`: The query to use for the test
+  - `expected::AbstractDict`: Expected values in the form
+    `Dict("/:output/:a/Int64" => [1, 2])` or
+    `Dict(:a => p1, 2])`
+    Keys can be symbols, which are mapped to /:output/:[symbol] and type derived from the
+    values, or a type that can be converted to string and used as relation path.
+  - `expected_problems::Vector`: expected problems. The semantics of
+    `expected_problems` is that the program must contain a super set of the specified
+    error codes.
+  - `allow_unexpected::Symbol`: ignore problems with severity equal or lower than
+    specified. Accepted values are `:none`, `:warning`, `:error`.
+  - `install::Dict{String, String}`: source files to install in the database.
+  - `schema_inputs::AbstractDict`: input schema for the transaction
+  - `inputs::AbstractDict`: input data to the transaction
```

Comment:
are these fields there for secret backwards compatibility, or are they truly things we want as part of the public API?

Reply:
`schema_inputs` is a little pointless since it's handled the same as `inputs` here, so 'secret backwards compatibility'? There are only 39 internal uses of it in cloud tests, so I think we can drop this with a tiny bit of extra work.
`inputs` are not necessary, and unliked style-wise, but they do make tests more flexibles and easy to reuse.

Reply:
sounds like maybe take `schema_inputs` out of the docstring so it'll be easier to yoink out later?

Reply:
Done.